starter-code/models/place.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PlaceSchema = new Schema({
+  name: String,
+  type: String,
+  location: {
+    type: String,
+    coordinates: [Number]
+  }
+});
+
```

Comment:
Indexing allows you to make faster querys by some field. In this case, you will do a lot of geographic searches, so it's a good idea to add after the schema:
```
placeSchema.index({ location: '2dsphere' });
```